easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'PythonBundle'
+
+name = 'CIRI-long'
+version = '1.0.2'
+
+homepage = 'https://github.com/bioinfo-biols/CIRI-long/'
+description = "Circular RNA Identification for Long-Reads Nanopore Sequencing Data"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('scikit-learn', '0.23.2'),
+    ('Pysam', '0.16.0.1'),
+    ('minimap2', '2.18'),
+    ('Cython', '0.29.22'),
+    ('SAMtools', '1.11'),
+    ('Biopython', '1.78'),
+    ('pyspoa', '0.0.8'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('mappy', '2.17', {
+        'checksums': ['ed1460efc9c6785df28065b7e93e93c92227f623a181f1a852dca6e6acb1a15f'],
+    }),
+    ('python-Levenshtein', '0.12.1', {
+        'modulename': 'Levenshtein',
+        'checksums': ['554e273a88060d177e7b3c1e6ea9158dde11563bfae8f7f661f73f47e5ff0911'],
+    }),
+    ('bwapy', '0.1.4', {
+        'checksums': ['fcdad0c7311753d550b3a3aefbc2619380e7e35e7aeaa386fd491cba47e71824'],
+    }),
+    ('nose', '1.3.7', {
```

Comment:
@zarybnicky This is already included with `Python` itself, so can be removed here.

Reply:
I'll submit an update PR